.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
You can remove this file from your git using git reset settings.json or something like that.